templates/taxi/car_detail.html

```diff
@@ -1,22 +1,22 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
+  <h1>Car Detail</h1>
+  <p><strong>Model:</strong> {{ car.model }}</p>
+  <p><strong>Manufacturer:</strong> {{ car.manufacturer.name }}</p>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
+  <h2>Drivers</h2>
   <ul>
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+
+  <form method="post">
+    {% csrf_token %}
+    <input type="hidden" name="car_id" value="{{ car.id }}">
+    <button type="submit" name="action" value="{% if request.user in car.drivers.all %}remove{% else %}add{% endif %}" class="btn {% if request.user in car.drivers.all %}btn-danger{% else %}btn-success{% endif %}">
```

Comment:
Refactor this so you don't have to repeat this block `if request.user in car.drivers.all`